### PR TITLE
Incident Timeline Support (Part I)

### PR DESCRIPTION
**Pull request by mvilanova**
This PR adds the model, service, and tests.

### CODE REVIEW

File: src/dispatch/event/models.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+from sqlalchemy import Column, Integer, String, ForeignKey, PrimaryKeyConstraint, Table
+from sqlalchemy.orm import relationship
+from sqlalchemy_utils import TSVectorType
+
+from dispatch.database import Base
+from dispatch.models import DispatchBase, TimeStampMixin
+from dispatch.plugins.base import plugins
+
+
+EventSource = Enum(
```
Comment (mvilanova) [inline src/dispatch/event/models.py]: Think about other sources and add them to the Enum.

Comment (kevgliss) [reply src/dispatch/event/models.py]: Why are these enums? Why can't we just use the slug strings themselves? Likely, we will have many different plugins adding things as events and will not want to manage enums.

Comment (mvilanova) [reply src/dispatch/event/models.py]: It's only one `Enum`. The idea was to have a structure with all the different sources of events that we can use in different parts of the code.

Comment (kevgliss) [reply src/dispatch/event/models.py]: If it's only one then I think a CONSTANT is better than an enum.

**Pull request merged**